search/services/index/results.py

```diff
@@ -92,13 +78,13 @@ def to_documentset(
     """
     max_pages = int(MAX_RESULTS / query.size)
     n_pages_raw = response["hits"]["total"] / query.size
-    n_pages = int(floor(n_pages_raw)) + int(n_pages_raw % query.size > 0)
-    logger.debug("got %i results", response["hits"]["total"])
+    n_pages = int(floor(n_pages_raw)) + int(int(n_pages_raw) % int(query.size) > 0)
```

Comment:
Is the additional casting to deal with mypy problems?

Reply:
Yes, I am really not sure why this was happening but this seemed to fix it. The mocking is a little odd ball.